TESTING.md

```diff
@@ -249,6 +249,41 @@ cargo make watch-test
 
 **Note:** For more information on publishing AMIs see [publishing](PUBLISHING.md).
 
+### vmware-k8s
+First, a management cluster needs to be created using [`EKS Anywhere`](https://github.com/aws/eks-anywhere).
```

Comment:
```suggestion
First, an initial management cluster needs to be created using [`EKS Anywhere`](https://anywhere.eks.amazonaws.com/docs/getting-started/production-environment/vsphere-getstarted/#create-an-initial-cluster).
```

Maybe also mention setting a variable to point to the management cluster's kubeconfig file and we can pass that below in `cargo make test`